cypress/e2e/0-JoinBet/registration/phone/phoneRegistration.cy.js

```diff
@@ -0,0 +1,55 @@
+import {bonusModalClose} from "../../../../support/utils/footer";
+import {header} from "../../../../fixtures/selectors";
+import langsHeader from '../../../../fixtures/translations/navigation/header';
+import {regByPhone} from "../../../../support/utils/reg";
+import {intermCheckReg} from "../../../../support/utils/reg";
+import {checkRegByPhone} from "../../../../support/utils/reg";
+
+describe('Registration by phone. Localization. Casino. Header.',() => {
+    langsHeader.forEach(({lang, headerCurrency}) => {
+        /*
+        C1447 - https://jbtest.testrail.io/index.php?/cases/view/1447
+         */
+        it(`Проверить список валют в дропдауне "Валюта" - язык ${lang}`, () => {
+            cy.authenticateUrl(`https://st2-gdjsmns.hk/${lang}/casino/lobby`);
+            bonusModalClose();
+            cy.get(header.regButton).click();
+            cy.get(header.window1.regChoiceButton.regEmailButton).click();
```

Comment:
@ElKalinin не поменял
```suggestion
            cy.get(header.window1.regChoiceButton.regPhoneButton).click();
```